### PR TITLE
[Test-3557][API] full cover DataAnalysisServiceImpl

### PR DESCRIPTION
**Pull request by SunJiFengPlus**
## What is the purpose of the pull request
#3557

## Brief change log
- full cover ```DataAnalysisServiceImpl```
- simplify ```DataAnalysisServiceImpl```  internal method
- delete try catch that will not throw any exeception

## Verify this pull request
截屏2020-08-28 15 36 46

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataAnalysisServiceImpl.java [hunk 1]
```diff
@@ -311,59 +265,9 @@
...
-                    projectIds,
-                    tasksQueueIds);
-        }
-
-        if (tasksKillIds.length != 0) {
-            taskKillCount = taskInstanceMapper.countTask(projectIds, tasksKillIds);
-        }
-
-        dataMap.put("taskQueue", taskQueueCount);
-        dataMap.put("taskKill", taskKillCount);
```
Comment (yangyichao-mango) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataAnalysisServiceImpl.java:365]: Will it produce error when we just use 
`dataMap.put("taskQueue", 0);`
`dataMap.put("taskKill", 0);`
?

**Review by yangyichao-mango — CHANGES_REQUESTED** [review]

**Comment by SunJiFengPlus** [comment]
Should i split the PR? It looks like no one has reviewed for a long time.

- PR1: full cover ```DataAnalysisServiceImpl``` & delete try catch that will not throw any exeception
- PR2: simplify ```DataAnalysisServiceImpl``` internal method

**Review by CalvinKirs — APPROVED** [review]
+1

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataAnalysisServiceImpl.java (hunk 1 above)
Comment (dailidong) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/DataAnalysisServiceImpl.java:365]: yes, it need to add detail data info , TODO

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**